docs/application/web/guides/data/file-system.md

```diff
@@ -108,149 +106,255 @@ To manage file storages:
+   ```
+   fileHandleRead.close();
+   ```
+ - You can copy, move and rename files and directories within the file system with the `copyFile()`, `copyDirectory()`, `moveFile()`, `moveDirectory()` and `rename()` methods. During copy or move operations, if a file or directory of the same name already exists in the target location, the overwrite input parameter of the method defines whether the existing file is overwritten.
+
+1. To copy a file, use the `copyFile()` method. The following example copies the file to the `images` directory. Since the third parameter is set to `true`, any existing file or directory with the same name in the target directory are overwritten.
 
    ```
-   var files; /* Assume that this is an array of File objects */
+   var filePathAndName = 'documents/exampleFile.jpg';
+   var destination = 'images';
    function onSuccess() {
        console.log('success');
    }
+   var errorCallback = function(error) {
+       console.log(error);
+   }
+   tizen.filesystem.copyFile(filePathAndName, destination, true, onSuccess, errorCallback);
+   ```
+
+2. To copy a directory, use the `copyDirectory()` method. The following example copies the directory to the `documents/` directory. Since the third parameter is set to `true`, any existing file or directory with the same name in the target directory are overwritten.
 
-   for (var i = 0; i < files.length; i++) {
-       documentsDir.copyTo(files[i].fullPath, 'images/backup/' + files[i].name,
-                           true, onSuccess);
+   ```
+   var directoryPathAndName = 'downloads/exampleDirectory';
+   var destination = 'documents';
+   function onSuccess() {
+       console.log('success');
    }
+   var errorCallback = function(error) {
+       console.log(error);
+   }
+   tizen.filesystem.copyDirectory(directoryPathAndName, destination, true, onSuccess, errorCallback);
+   ```
+
+3. To move a file, use the `moveFile()` method. The following example moves the files to the `music/` directory. Since the third parameter is set to `false`, no existing files with the same name in the target directory are overwritten.
+
+   ```
+   var filePathAndName = 'downloads/exampleFile.mp3';
+   var destination = 'music';
+   function onSuccess() {
+       console.log('success');
+   }
+   var errorCallback = function(error) {
+       console.log(error);
+   }
+   tizen.filesystem.moveFile(filePathAndName, destination, true, onSuccess, errorCallback);
    ```
 
-4. To move a file or directory, use the `moveTo()` method. The following example moves the files to the `images/newFolder/` directory. Since the third parameter is set to `false`, no existing files with the same name in the target directory are overwritten.
+4. To move a directory, use the `moveDirectory()` method. The following example moves the directory to the `camera/` directory. Since the third parameter is set to `true`, any existing file or directory with the same name in the target directory are overwritten.
```

Comment:
Since the third parameter is set to `true`, <all files or directories with the same name existing> in the target directory are overwritten.

I think that files will be overwritten - 'any' means an opposite case.